From 4aa512dfeab609aa539bb9e262cbc7bff2ce6a49 Mon Sep 17 00:00:00 2001
From: Lily Zhang <r58066@freescale.com>
Date: Mon, 12 Mar 2012 21:36:28 +0800
Subject: ENGR00176812-8 mx6 sabreauto: add board_is_mx6_xxx macro

Add the following macros to make codes readable.
board_is_mx6_sabreauto_reva()
board_is_mx6_sabreauto_revb()

Signed-off-by: Lily Zhang <r58066@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/plat-mxc/include/mach/mxc.h b/arch/arm/plat-mxc/include/mach/mxc.h
index c8f1805..caa218b 100755
--- a/arch/arm/plat-mxc/include/mach/mxc.h
+++ b/arch/arm/plat-mxc/include/mach/mxc.h
@@ -92,6 +92,10 @@ extern unsigned int system_rev;
 	board_is_rev(IMX_BOARD_REV_2))
 #define board_is_mx6q_sabre_auto() (cpu_is_mx6q() && \
 	board_is_rev(IMX_BOARD_REV_3))
+#define board_is_mx6_sabreauto_reva() \
+	board_is_rev(IMX_BOARD_REV_2)
+#define board_is_mx6_sabreauto_revb() \
+	board_is_rev(IMX_BOARD_REV_1)
 #endif
 
 #ifndef __ASSEMBLY__
-- 
1.7.9.5

